src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Android.cs

```diff
@@ -0,0 +1,400 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System
+{
+    public sealed partial class TimeZoneInfo
+    {
+        private const string TimeZoneFileName = "tzdata";
+
+        private static AndroidTzData? tzData;
+        private static readonly object tzDataLock = new object();
+
+        private static AndroidTzData atzData
```

Comment:
I'd name this something like AndroidTzDataInstance.  Whatever you decide, use PascalCasing for properties (all methods basically).

Reply:
Updated!